js/actions/api-pregel.js

```diff
@@ -62,18 +62,18 @@ function post_api_pregel (req, res) {
   if (!json.algorithm || typeof json.algorithm !== 'string') {
```

Comment:
can't we delete this file now ?

Reply:
Yes, we can. I thought I had deleted it already, but clearly I was mistaken.